qa/integration/specs/reload_config_spec.rb

```diff
@@ -84,7 +84,14 @@
     pipeline_event_stats = logstash_service.monitoring_api.pipeline_stats("main")["events"]
     expect(pipeline_event_stats["in"]).to eq(1)
     expect(pipeline_event_stats["out"]).to eq(1)
-    
+
+    # make sure the pipeline is not facing a backpressure and has zero input throughput when reloaded
+    pipeline_flow_stats = logstash_service.monitoring_api.pipeline_stats("main")["flow"]
+    expect(pipeline_flow_stats["input_throughput"]["lifetime"]).to eq(0)
+    expect(pipeline_flow_stats["input_throughput"]["current"]).to eq(0)
+    expect(pipeline_flow_stats["queue_backpressure"]["lifetime"]).to eq(0)
+    expect(pipeline_flow_stats["queue_backpressure"]["current"]).to eq(0)
```

Comment:
Nit(feel free to defer to another PR): Arguably safer and should give us a better view of what the whole shape was in event of failure:

```suggestion
    expect(pipeline_flow_stats).to include({
      "input_throughput"   => hash_including("current" => 0, "lifetime" => 0),
      "queue_backpressure" => hash_including("current" => 0, "lifetime" => 0),
    })
```